src/mono/sample/wasm/console-v8/main.mjs

```diff
@@ -3,7 +3,7 @@
 
 import { dotnet } from './dotnet.js'
 
-dotnet
+await dotnet
```

Comment:
This change seems like it could break BenchmarkDotNet, I believe it relies on our existing main files

Reply:
this `await` is optional